fpdf/html.py

```diff
@@ -817,6 +819,7 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         warnings.warn(
             (
+                # pylint: disable=implicit-str-concat,useless-suppression
```

Comment:
Since the strings are placed within their own set of parens, this directive is unnecessary (useless-suppression already tries to tell you that).

f("str1" "str2" "str3")  # ok: alone within argument list
f( ("str1" "str2" "str3"), other, args)  # ok: not alone, but with their own parens
f("str1" "str2" "str3", other, args)  # NOT ok: as part of a comma seperated list

The point is to make it explicit that you didn't forget any commas in between.

Reply:
I'm well aware of that, I once even contributed a similar check to Pylint 😄 https://github.com/pylint-dev/pylint/pull/1655

I still had a warning with pylint 2.17.5:
```
$ pylint --version
pylint 2.17.5
astroid 2.15.6
Python 3.8.10 (default, Nov 14 2022, 12:59:47)
[GCC 9.4.0]

$ pylint fpdf/html.py
************* Module fpdf.html
fpdf/html.py:822:0: W1404: Implicit string concatenation found in call (implicit-str-concat)
```

But upgrading to Pylint 3.0.2 got rid of the warning, so no need for this comment, I'm going to remove this line from this PR